prs.py

```diff
@@ -69,57 +75,95 @@ def tree_matmul_tree_matsum(bm1, bm2, mul_splits: int, sum_splits: int = None,
     sum_block_size = math.ceil(mul_splits/sum_splits)
     for i in range(sum_splits): #range(sum_splits):
         partial_sum_path = f"{path_prefix}-partial-{i}"
-        # sum(read_intermediates[i*sum_block_size:(i+1)*sum_block_size]).write(partial_sum_path)
+        sum(read_intermediates[i*sum_block_size:(i+1)*sum_block_size]).write(partial_sum_path, overwrite = overwrite)
         tracked_partial_sums.append(BlockMatrix.read(partial_sum_path))
         
     return sum(tracked_partial_sums)
+
+def get_pheno_id_w_desc(mt):
+    r'''
+    Add pheno_id column to mt using existing trait_type, phenocode, pheno_sex, coding, modifier, and description columns
+    '''
+    mt = mt.annotate_cols(pheno_id = (mt.trait_type+'-'+
+            mt.phenocode+'-'+
+            mt.pheno_sex+
+            ('-' + hl.if_else(pd.notnull(mt.coding) and mt.coding != '', mt.coding, ''))+
```

Comment:
Does this really work? I'm pretty sure this is always going to return True (how is pandas operating on a column field of a matrixtable?)